source/js/main.js

```diff
@@ -520,6 +520,44 @@ let main = {
 
       injectDonateModal(donationModal, modalOptions);
     }
+
+    // Enable the "load more results" button on index pages
+    let loadMoreButton = document.querySelector(`.load-more-index-entries`);
+    if (loadMoreButton) {
+      let entries = document.querySelector(`.index-entries`);
+
+      let page = 0;
+      let page_size = 1; // TODO: should we get this as data-attribute, set from IndexPage.page_size ?
+
+      const loadMoreResults = evt => {
+        // Construct our localized API url:
+        let url = `./entries/?page=${page++}&page_size=${page_size}`;
+
+        // And then fetch the results and render them into the page.
+        fetch(url)
```

Comment:
Do we need to do anything for Percy here?

Reply:
Hm, this would be a good candidate for true interaction testing, rather than percy. Unfortunately, we still don't have that quite finished and landed...